src/back/spv/writer.rs

```diff
@@ -724,9 +769,13 @@ impl Writer {
     fn parse_type_declaration(
         &mut self,
         arena: &crate::Arena<crate::Type>,
-        handle: crate::Handle<crate::Type>,
+        lookup_type: &LookupType<crate::Type>,
```

Comment:
I can change this to a value type, and deference it at the inserts in this function. Problem is that in `get_type_id` I get the "Cannot move" error.

Reply:
please tell more!